oauthproxy.go

```diff
@@ -793,6 +813,16 @@ func (p *OAuthProxy) OAuthCallback(rw http.ResponseWriter, req *http.Request) {
 	}
 	errorString := req.Form.Get("error")
 	if errorString != "" {
+		errorDescription := req.Form.Get("error_description")
+
+		ceb := p.findCallbackErrorRedirect(errorString, errorDescription)
+
```

Comment:
I've popped those together - I don't usually mix declaration, assignment and predicate into the same statement, is this a golang thing? I would have thought doing this would set the alarm bells ringng in the cognitive check! I'm having real problems with your cognitive check :(